install.rdf

```diff
@@ -6,7 +6,7 @@
     <em:hasEmbeddedWebExtension>true</em:hasEmbeddedWebExtension>
     <em:type>2</em:type>
     <em:unpack>false</em:unpack>
-    <em:version>1.0.10</em:version>
+    <em:version>0.4.0</em:version>
```

Comment:
Will this affect auto updates in any way since we're seemingly downgrading the version number from v1 to v0.4? Or should we just bump all version numbers to v2.0.0 and keep them in sync?